src/views/HighscoreView.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <h1>Highscore</h1>
```

Comment:
Hier sollten wir `h2` verwenden. 
In einem Projekt sollte es insgesamt nur eine `h1` geben.

Reply:
> Hier sollten wir `h2` verwenden. In einem Projekt sollte es insgesamt nur eine `h1` geben.

Pro Seite sollte es eine `h1` geben.

Auf einer klassischen Website wäre z.B. nicht der Firmennamen die `h1` sondern die jeweilige Seitenüberschrift.

Reply:
jetzt bin ich verwirrt. also hab ich die jetzt umsonst geändert? :D